README.md

```diff
@@ -127,10 +126,11 @@ knife-ec2 now includes the ability to retrieve the encrypted data bag secret and
 ```
 
 ### Supported URL format
-- `http` or `https` based: 'http://provisioning.bucket.com/chef/my-validator.pem'
+- `http` or `https` based: '[http://provisioning.bucket.com/chef/my-validator.pem](http://provisioning.bucket.com/chef/my-validator.pem)'
 - `s3` based:  's3://chef/my-validator.pem'
 
 ### Use the following configuration options in `knife.rb` to set the source URLs:
+
 ```ruby
 knife[:validation_key_url] = 'http://provisioning.bucket.com/chef/my-validator.pem'
 knife[:s3_secret] = 'http://provisioning.bucket.com/chef/encrypted_data_bag_secret'
```

Comment:
I'd probably change bucket.com to example.org all through here. it took me a little internal debate to be sure that provisioning.bucket.com wasn't a real thing.